internal/artifacts/module_arch.go

```diff
@@ -230,7 +230,7 @@ func checkBuildResultsConflicts(mtaObj *mta.MTA, source, target string, extensio
 			}
 			moduleName, pathInUse := resultPathModuleNameMap[targetArtifact]
 			if pathInUse {
-				return errors.Errorf(multiBuildWithPathsConflictMsg, module.Name, moduleName, targetArtifact)
+				return errors.Errorf(multiBuildWithPathsConflictMsg, module.Name, moduleName, moduleLoc.GetTarget())
```

Comment:
What does this output for example and what was the output for example from before this change?

Reply:
before it was like:
could not build the modules selected: could not save the build results of modules "n2" and "n1" in the "C:\Users\......\testpartialbuild\partialbuildtest1\data.zip" target folder because of conflicting naming; use the "build-artifact-name" build parameter to create a unique name for each module

now:
could not build the modules selected: could not save the build results of modules "n2" and "n1" in the "C:\Users\......\testpartialbuild\partialbuildtest1" target folder because of conflicting naming; use the "build-artifact-name" build parameter to create a unique name for each module

Reply:
I think it would be useful for the user to know the file name too, maybe the message itself should be changed or the file name added separately from the folder. Could you consult with Paola on how to add this information to the message?